app/src/main/java/cs465/illinois/edu/dogonthequad/MeetupInProgressActivity.java

```diff
@@ -0,0 +1,11 @@
+package cs465.illinois.edu.dogonthequad;
+
+import android.app.Activity;
+import android.os.Bundle;
+
+public class MeetupInProgressActivity extends Activity {
```

Comment:
This class doesn't seem very necessary, since the only thing it does is set the content view (which can be done in `CreateMeetupReviewActivity`).